docs/advanced-features/materialization.md

```diff
@@ -1,34 +1,35 @@
 <!--pytest-codeblocks:skipfile-->
 <!-- markdownlint-disable -->
 
-## `build` versus `push`
-
-`build` and `push` have the following behaviors:
-
-* `build` calculates a tophash based on the package entry data, package entry metadata, and top-level package metadata, injects that into the package manifest, and stores the package manifest in the local registry.
-* `push` uploads the entries in the package to a remote registry, performs the same tophash calculations, and stores a package manifest referencing the remote files in the remote registry.
-
-Under the hood, there are just two differences between `push` and `build`:
-
-* `push` targets a remote registry; `build` targets a local registry
-* `push` copies package files; `build` leaves files where they are
-
 ## Materialization
 
-The latter of these two differences is known as **materialization**, and it means that `push` creates **materialized packages**: packages which point solely to files located in an S3 bucket.
+`push` uploads the entries in a data package to a remote registry,
+performs the tophash calculations, and stores a package manifest
+referencing the remote files in the remote registry.
+
+This is known as **materialization**,
+and it means that `push` creates **materialized packages**: packages
+which point solely to files located in an Amazon S3 bucket.
 
-A materialized package is stronger than a unmaterialized (or "local") package because S3 guarantees that individual object versions are never lost or destroyed (assuming object versioning is enabled). Furthermore, only materialized packages may be browsed in the online Quilt catalog.
+A materialized package is powerful because Amazon S3 guarantees that
+individual object versions are never lost or destroyed (assuming
+object versioning is enabled).  Furthermore, only materialized
```

Comment:
> S3 guarantees that individual object versions are never lost or destroyed
This is not true. More like: "with appropriate IAM controls S3 object versions can be protected against accidental deletion"